### PR TITLE
users follow working, users unfollow working, users profile page list…

### PR DESCRIPTION
**Pull request by tanzeelrana**
users follow working, users unfollow working, users profile page list of followers is connected to the UI, users list of following is also connected to the UI, filters working on lists of followers/following on user profile page. Next need to connect pages and likes tab, also need to do something about the "All" tab. Need to figure out a easy way to edit user profile information, currently update profile not working. pagination also not working.

### CODE REVIEW

**Comment by mrakulous** [comment]
Looks good to me.  I'll let Haris give it a quick review as well before merging.

**Review by hrghauri — APPROVED** [review]
It looks good to me as well. We can merge it.

**Pull request merged**
